wgpu-core/src/instance.rs

```diff
@@ -560,17 +586,30 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             let (adapter_guard, _) = hub.adapters.read(&mut token);
             let adapter = &adapter_guard[adapter_id];
             let phd = &adapter.raw.physical_device;
+
+            let actual_features = adapter.raw.physical_device.features();
```

Comment:
Let's work on the naming here a bit. I think this should be `available_features`.
Then we can have `enabled_features` for stuff that the user requested, like `SAMPLER_ANISOTROPY`.
And finally, we can have `actual_features` as a result of all of that that we actually pass down.